.github/workflows/tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: >
```

Comment:
This block is meant to prevent a double-test run at a pull request...but it's not quite working as expected...it's running the test twice in this case still. We can fix that next, it's not a big time sink.